draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -475,11 +476,21 @@ The hierarchical components' identifier could be found by the "component-referen
 According to the management requirements from operators, some important attributes are not defined in {{!RFC8348}}. These attributes could be component-specific and are not suitable to define under the component list node. So, the model can be augmented with HW/SW specific-info-grouping containing attributes applicable to HW e.g. boards/slot or SW e.g. software-module/boot-loader component only.
 
 ~~~~ ascii-art
-  augment /ni:network-elements/ni:network-element:
-    +--rw virtual-ne-attributes
-  augment /ni:network-elements/ni:network-element/ni:components
-            /ni:component:
-    +--rw software-module-specific-info
++--rw components
+   +--rw component* [component-id]
+   |  +--rw component-id            string
+   .......................................
+   +--rw (component-class)?
+      +--:(chassis)
+      |  +--rw chassis-specific-info
+      +--:(container)
+      |  +--rw slot-specific-info
+      +--:(module)
+      |  +--rw board-specific-info
+      +--:(port)
+      |  +--rw port-specific-info
+      +--:(software)
+         +--rw software-specific-info
```

Comment:
The `component-class` is at the same level as `component-id`:

```suggestion
+--rw components
   +--rw component* [component-id]
      +--rw component-id            string
      .......................................
      +--rw (component-class)?
         +--:(chassis)
         |  +--rw chassis-specific-info
         +--:(container)
         |  +--rw slot-specific-info
         +--:(module)
         |  +--rw board-specific-info
         +--:(port)
         |  +--rw port-specific-info
         +--:(software)
            +--rw software-specific-info
```

Reply:
Thank you Italo, now it is moved to the same level with component-id.